From 2b6e77ff3adcb08c2deed1819727d918499df031 Mon Sep 17 00:00:00 2001
From: Markus Niebel <Markus.Niebel@tq-group.com>
Date: Thu, 14 Feb 2019 16:41:09 +0100
Subject: [PATCH] tqma8qx-mba8qx: update defconfig

CMD_NET needed for functional network

while at it, bring phylib, GIGE PHY support and TI Phy to defconfig

Signed-off-by: Markus Niebel <Markus.Niebel@tq-group.com>
---
 configs/tqma8qx_mba8qx_mmc_defconfig | 6 ++++--
 include/configs/tqma8qx-mba8qx.h     | 6 +++---
 include/configs/tqma8qx.h            | 2 +-
 3 files changed, 8 insertions(+), 6 deletions(-)

diff --git a/configs/tqma8qx_mba8qx_mmc_defconfig b/configs/tqma8qx_mba8qx_mmc_defconfig
index f081f76fc7..2ebbe877cb 100644
--- a/configs/tqma8qx_mba8qx_mmc_defconfig
+++ b/configs/tqma8qx_mba8qx_mmc_defconfig
@@ -19,13 +19,12 @@ CONFIG_CMD_I2C=y
 # CONFIG_CMD_LOADS is not set
 CONFIG_CMD_MMC=y
 CONFIG_CMD_SF=y
-# CONFIG_CMD_NET is not set
-# CONFIG_CMD_NFS is not set
 CONFIG_CMD_CACHE=y
 CONFIG_CMD_EXT4_WRITE=y
 # CONFIG_ISO_PARTITION is not set
 # CONFIG_BLK is not set
 CONFIG_DM_GPIO=y
+CONFIG_MXC_GPIO=y
 CONFIG_DM_I2C=y
 CONFIG_SYS_I2C_IMX_LPI2C=y
 CONFIG_DM_MMC=y
@@ -38,7 +37,10 @@ CONFIG_SPI_FLASH_STMICRO=y
 CONFIG_SPI_FLASH_WINBOND=y
 # CONFIG_SPI_FLASH_USE_4K_SECTORS is not set
 CONFIG_SPI_FLASH_MTD=y
+CONFIG_PHYLIB=y
+CONFIG_PHY_TI=y
 CONFIG_DM_ETH=y
+CONFIG_PHY_GIGE=y
 CONFIG_PINCTRL=y
 CONFIG_PINCTRL_IMX8=y
 CONFIG_POWER_DOMAIN=y
diff --git a/include/configs/tqma8qx-mba8qx.h b/include/configs/tqma8qx-mba8qx.h
index 685343e232..bd82fb65f3 100644
--- a/include/configs/tqma8qx-mba8qx.h
+++ b/include/configs/tqma8qx-mba8qx.h
@@ -11,9 +11,9 @@
 #define CONFIG_FEC_XCV_TYPE		RGMII
 #define FEC_QUIRK_ENET_MAC
 
-#define CONFIG_PHY_GIGE /* Support for 1000BASE-X */
-#define CONFIG_PHYLIB
-#define CONFIG_PHY_TI
+/* #define CONFIG_PHY_GIGE */ /* Support for 1000BASE-X */
+/* #define CONFIG_PHYLIB */
+/* #define CONFIG_PHY_TI */
 #define CONFIG_MII
 
 #define MX8QX_FEC1_BASE			0x5B040000U
diff --git a/include/configs/tqma8qx.h b/include/configs/tqma8qx.h
index b339dcd0ca..b9ba12350a 100644
--- a/include/configs/tqma8qx.h
+++ b/include/configs/tqma8qx.h
@@ -40,7 +40,7 @@
 #define CONFIG_CMD_FUSE
 
 /* GPIO configs */
-#define CONFIG_MXC_GPIO
+/* #define CONFIG_MXC_GPIO */
 
 #ifdef CONFIG_AHAB_BOOT
 #define AHAB_ENV "sec_boot=yes\0"
